utils/cliutils/commandsflags.go

```diff
@@ -1205,11 +1205,15 @@ var flagsMap = map[string]cli.Flag{
 	},
 	Npm: cli.BoolFlag{
 		Name:  Npm,
-		Usage: "[Optional] Request audit for a npm project.` `",
+		Usage: "[Optional] Request audit for an npm project.` `",
+	},
+	Yarn: cli.BoolFlag{
+		Name:  Yarn,
+		Usage: "[Optional] Request audit for a Yarn project.` `",
```

Comment:
...for a Yarn project... --> ...for a Yarn 2 project...

Reply:
Changed to:
...for a Yarn 2+ project...